graylog2-server/src/main/java/org/graylog/grn/GRNRegistry.java

```diff
@@ -33,7 +33,8 @@
  */
 @Singleton
 public class GRNRegistry {
-    public static final String GLOBAL_USER_GRN = "grn::::builtin-team:everyone"; // TODO: Find a better name for the "everyone" grantee GRN type
+    // TODO: Find a better name for the "everyone" grantee GRN type
+    public final GRN GLOBAL_USER_GRN = newGRN(GRNTypes.BUILTIN_TEAM, "everyone");
```

Comment:
How about using this? That way we can make it `static` and don't have a public instance field that looks like a constant.
 
```suggestion
    public static final GRN GLOBAL_USER_GRN = BUILTIN_TEAM.newGRNBuilder().entity("everyone").build();;
```

Reply:
ah, much nicer